tools/doc/html.js

```diff
@@ -416,15 +416,15 @@ const BSD_ONLY_SYSCALLS = new Set(['lchmod']);
 // '<a href="http://man7.org/linux/man-pages/man2/open.2.html">open(2)</a>'
 function linkManPages(text) {
   return text.replace(
-    /\b([a-z.]+)\((\d)([a-z]?)\)/gm,
-    (match, name, number, optionalCharacter) => {
+    /(^|\s)([a-z.]+)\((\d)([a-z]?)\)/gm,
+    (match, beginning, name, number, optionalCharacter) => {
       // name consists of lowercase letters, number is a single digit
-      const displayAs = `${name}(${number}${optionalCharacter})`;
+      const displayAs = `${beginning}${name}(${number}${optionalCharacter})`;
       if (BSD_ONLY_SYSCALLS.has(name)) {
-        return ` <a href="https://www.freebsd.org/cgi/man.cgi?query=${name}` +
+        return `<a href="https://www.freebsd.org/cgi/man.cgi?query=${name}` +
```

Comment:
I think `beginning` should be used here, replacing the space.

Reply:
@lpinca Same on line 427 too? Or am I missing something?

Reply:
yes, that's correct.